docker/Dockerfile.omp_offload

```diff
@@ -0,0 +1,110 @@
+##############################################################
```

Comment:
I'd call this file `Dockerfile.nvidia.clang` 

And probably I'd rename the other one (`Dockerfile.nvidia`) as `Dockerfile.nvidia.nvc`

@mloubout @kenhester 

Ideally, they would inherit from a common `Dockerfile.nvidia`


Food for thought

Reply:
Alright. Will reorganize the Dockerfiles in the next commit.